app/pubspec.yaml

```diff
@@ -45,4 +45,4 @@ dev_dependencies:
 dependency_overrides:
   # Fixes https://github.com/dart-lang/pana/issues/138
   # Required until we have a 'gold' release
-  analyzer: 0.31.0-alpha.2
+  analyzer: 0.31.1
```

Comment:
Can't you just remove the dependency_override entirely, @isoos ?

Reply:
Only after we've upgraded with the appropriate pana version (it will be in 0.10.2).